airbyte-integrations/connectors/destination-selectdb/src/main/java/io/airbyte/integrations/destination/selectdb/SelectdbDestination.java

```diff
@@ -0,0 +1,170 @@
+    try {
+      final Path destinationDir = getTempPathDir(config);
+      FileUtils.forceMkdir(destinationDir.toFile());
+      for (ConfiguredAirbyteStream stream : configuredCatalog.getStreams()) {
+
+        final DestinationSyncMode syncMode = stream.getDestinationSyncMode();
+        if (syncMode == null) {
+          throw new IllegalStateException("Undefined destination sync mode");
+        }
+
+        final String streamName = stream.getStream().getName();
+        final String tableName = namingResolver.getIdentifier(streamName);
+        final String tmpTableName = namingResolver.getTmpTableName(streamName);
+        final Path tmpPath = destinationDir.resolve(tmpTableName + ".csv");
+        if (conn == null)
+          checkSelectdbAndConnect(config);
+        Statement stmt = conn.createStatement();
+        stmt.execute(createTableQuery(tableName));
+        if (syncMode == DestinationSyncMode.OVERWRITE) {
+          stmt.execute(truncateTable(tableName));
+        }
+        CSVFormat csvFormat = CSVFormat.DEFAULT
+                .withSkipHeaderRecord()
+                .withDelimiter(SelectdbCopyInto.CSV_COLUMN_SEPARATOR)
+                .withQuote(null)
+                .withHeader(
+                        JavaBaseConstants.COLUMN_NAME_AB_ID,
+                        JavaBaseConstants.COLUMN_NAME_EMITTED_AT,
+                        JavaBaseConstants.COLUMN_NAME_DATA);
+        final FileWriter fileWriter = new FileWriter(tmpPath.toFile(), Charset.defaultCharset(), false);
+        final CSVPrinter printer = new CSVPrinter(fileWriter, csvFormat);
+        SelectdbCopyInto sci = new SelectdbCopyInto(
+                tmpPath,
+                SelectdbConnectionOptions.getSelectdbConnection(config, tableName),
+                new LabelInfo("",tableName),
+                http.getClient(),
+                JavaBaseConstants.COLUMN_NAME_AB_ID,
+                JavaBaseConstants.COLUMN_NAME_EMITTED_AT,
+                JavaBaseConstants.COLUMN_NAME_DATA);
+        writeConfigs.put(streamName, new SelectdbWriteConfig(sci, printer, csvFormat));
+      }
+    } catch (SQLException | ClassNotFoundException e) {
+      LOGGER.error("Exception while creating Selectdb destination table: ", e);
+      throw new SQLException(e);
+    } catch (IOException e) {
+      LOGGER.error("Exception while handling temporary csv files : ", e);
+      throw new IOException(e);
+    } finally {
+      if (conn != null)
+        conn.close();
+    }
+    return new SelectdbConsumer(writeConfigs, configuredCatalog, outputRecordCollector);
+  }
+
+
+  protected void checkSelectdbAndConnect(JsonNode config) throws ClassNotFoundException, SQLException {
+    SelectdbConnectionOptions selectdbConnection = SelectdbConnectionOptions.getSelectdbConnection(config, "");
+    String dbUrl = String.format(DB_URL_PATTERN, selectdbConnection.getJdbcUrl(), selectdbConnection.getDb());
+    Class.forName(JDBC_DRIVER);
```

Comment:
why is this needed? are you truing to load the jdbc driver explicitly?

Reply:
If you mean that the connection pool should be used here, I don't think it is necessary, because the connection here is only to verify whether the link configuration is available and execute the create table and TRUNCATE statements infrequently. If it's not a connection pool issue, I may not understand what you mean.